README.md

```diff
@@ -42,7 +40,7 @@ Anybody working for the Financial Times is welcome to suggest changes via a GitH
 The Engineering competencies are used to help identify when an engineer is ready for promotion. Every engineer in the CTO organisation of the Financial Times can help shape these competencies. While they are in alpha and beta we are especially interested in contributions.  [A guide for adding or editing competencies is here](docs/competencies.md).
 
   - **Contribute to the documentation and website**<br/>
-    It's important that the documentation in this repo and the Engineering Progression website are clear and easy to understand. [Get started maintaining the documentation](docs/documentation.md)
+    It's important that the documentation in this repo and the Engineering Progression website is clear and easy to understand. [Get started maintaining the documentation](docs/documentation.md)
```

Comment:
the subject of this sentence is `the documentation`, which is a singular mass noun